models/intermediate/int_iterable__list_user_history.sql

```diff
@@ -26,7 +26,7 @@ with user_history as (
         updated_at
 
     from previous_email_list_ids
-    where email_list_ids != previous_ids -- list ids are always stored in their arrays in numerical order
+    where email_list_ids != coalesce(previous_ids, 'this is new') -- list ids are always stored in their arrays in numerical order
```

Comment:
Could this cause any errors since the ids are usually numerical? I see in the source we specify the [macro](https://github.com/fivetran/dbt_iterable_source/blob/4c45f69ab4b62e7e2b2fe727bf300381e026b95b/macros/get_user_history_columns.sql#LL6C63-L6C63) as type string. Could you just confirm that in cases where this is used it does not cause a datatype mismatch error.

Reply:
Great! If it will always be an array then I think we can be comfortable that this will always be a string. No need to cast and thanks for the explenation!